src/routes/(protected)/calendar/+page.svelte

```
@@ -136,23 +228,116 @@
+            const newMonth = direction === "prev" ? (currentDate.month === 1 ? 12 : currentDate.month - 1) : (currentDate.month === 12 ? 1 : currentDate.month + 1);
+            const newYear = direction === "prev" ? (currentDate.month === 1 ? currentDate.year - 1 : currentDate.year) : (currentDate.month === 12 ? currentDate.year + 1 : currentDate.year);
+            target = new CalendarDate(newYear, newMonth, 1);
         }
+        const weekSunday = viewMode === "weekly" ? dateUtils.addDays(target, -dateUtils.getDayOfWeek(target)) : target;
+        const url = buildUrl({
+            view: viewMode === "monthly" ? null : viewMode,
+            year: String(weekSunday.year),
+            month: String(weekSunday.month),
+            day: viewMode === "daily" ? String(target.day) : null
+        });
+        goto(url, { keepFocus: true }).catch(() => {}).finally(() => { isNavigating = false; });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)